aws/resource_aws_ecs_service.go

```diff
@@ -56,6 +56,12 @@ func resourceAwsEcsService() *schema.Resource {
 				},
 			},
 
+			"enable_ecs_managed_tags": {
```

Comment:
should this be setting `ForceNew: true`? From what I can see, this property is only configurable during the initial [creation of a service](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_CreateService.html) and not for [`UpdateService`](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_UpdateService.html) API. If I try to flip it on for an existing service, TF plan repeatedly tries to re-set it back to true because the updates aren't going through.